example/demo.html

```diff
@@ -3,10 +3,8 @@
     <head>
         <meta charset="utf-8"/>
         <title>Inline template test</title>
-        <link href="https://fonts.googleapis.com/css?family=Mada:400,500&display=swap" rel="stylesheet">
-        <link href="https://fonts.googleapis.com/css2?family=Pathway+Gothic+One&display=swap" rel="stylesheet">
-        <link rel="stylesheet" href="https://use.typekit.net/bbh8fwb.css">
-        <link rel="stylesheet" href="https://use.typekit.net/xwu6uop.css">
+        <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
``preconnect`` needs the ``crossorigin`` attribute to work properly.

Reply:
Haven't been able to find any documentation on why they have as is, but that's exactly how the google font website tells you to put it:
![image](https://user-images.githubusercontent.com/54490191/99484748-148e3d80-292f-11eb-9701-6c2fd034c56c.png)
I think it's a relatively new change because when I last added a google font to a project it didn't include the preconnect at all.

Reply:
Google may not explicitly want you to use it, but we should because it can decrease load times by roughly 100ms. 

I can't say for sure why Google isn't just adding it themselves, but this [article](https://www.cdnplanet.com/blog/faster-google-webfonts-preconnect/) is from 2017 and explains it pretty well. 

Basically if you only do ``preconnect`` it will only do the DNS lookup and not actually connect and get things ready. If you add ``crossorigin`` it will do DNS/TLS/TCP all in parallel with the CSS being loaded.

``<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>`` <---- Fast Mode
``<link rel="preconnect" href="https://fonts.gstatic.com"> <---- Slow Mode

Reply:
Okay, will update this part later today. 

I think the cover image is still TBD?
For consistency/accuracy sake I'd prefer to not attempt to recreate it in css. But I think nono was still noodling over how to approach it.

Reply:
I mean if you base64 encode it like Ay had said then bam it's recreated in CSS and it looks exactly the same 1:1 and it will be baked into the library meaning no reliance on third party hosting. There is pretty much no reason to not do the encoded approach for this use case. Which is what I believe Nono was alluding to when he said to inline it, I can't speak for him but the point being that if its baked into the lib it is fundamentally better than relying on a third party hosting it.

Reply:
Okay! But I don't know how to do that (or really what it means other than breaking down the image into its data?).